submissions/JOSEF/Alt-F4 - JOSEF.md

```diff
@@ -0,0 +1,156 @@
+
+The builder train in charge simply alternates between the two different cells. There’s a simple “inserter cycle” that takes the blueprint book out of the train when it gets home, puts in the other one, then waits for the train to leave before activating again.
+
+![Production cell 1: Basic, non-oily things](https://i.imgur.com/BccZ1d9.png)
+
+The first cell produces all the different basic building materials: Belts, assemblers, miners, inserters, rails, trains, stations, signals, power poles – more or less everything that doesn‘t require oil. The basic production cell needs to be built before JOSEF runs out of stuff like belts or rails. This may be a problem if you have too many resource patches in the center or if you don’t give it a enough to start with and it decides to just solar panel the world. But you can always fix this by giving it a few more starting materials.
+
+  
+
+![Production cell 2: Advanced things using oil](https://i.imgur.com/CQsUf42.png)
+
+The second cell produces the more advanced things: Roboports, robots, logistic chests, blueprint deployers, solar panels & accumulators. This one needs to be built before JOSEF runs out of roboports. The builder train brings in 10 barrels of heavy oil which get emptied, run through the refinery and get refilled as soon as possible. JOSEF is not producing barrels but instead relying on the initial ones getting recycled infinitely.
+
+The two production cells are far from optimized, built to ratio or anything like that. They slowly and steadily produce all the things necessary but it may take an hour until a certain cell produces, say, their first train station, because it needs to fill up with belts first. However, it turns out that this becomes a non-issue after a bit, but more on that later.
+
+How many of these cells actually get supplied with ores depends on available resources, of course. Trains will always prefer to feed those cells close to resource patches so some further away cells might just stay empty for a while. But they will sooner or later get their resources, simply because there is an incredible number of...
+
+## ...Trains
+
+The number of trains gets ridiculous very quickly. In my >500 cell run JOSEF was using 750 trains. The reason: For every new production cell that gets built, 4 new trains (one for each ore) get deployed in a dedicated “train deployer” cell in the center. Most of them will sit in a station and wait to be unloaded, so that's not a problem.
+
+Deploying trains in an extra cell (instead of including them in the production blueprints) ensures that trains get deployed in a position where they don't interfere with existing trains, which can cause the new ones to attach to existing trains. 
+
+## General & Outlook
```

Comment:
Part of this section you could describe how Josef performed. How big could you get him? Are you still running him? What kind of UPS did you get after how every many hours?